pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -55,6 +55,8 @@ func init() {
 			rbac.NewRule("get").Groups(imageGroup, legacyImageGroup).Resources("imagestreams").RuleOrDie(),
 			rbac.NewRule("get", "list", "create", "delete").Groups(kapiGroup).Resources("pods").RuleOrDie(),
 			rbac.NewRule("get").Groups(kapiGroup).Resources("namespaces").RuleOrDie(),
+			rbac.NewRule("get").Groups(buildGroup, legacyBuildGroup).Resources("secrets").RuleOrDie(),
+			rbac.NewRule("get").Groups(buildGroup, legacyBuildGroup).Resources("configmaps").RuleOrDie(),
```

Comment:
@deads2k we need this so the controller can resolve env var references to secrets/configmaps.  any concerns?